docs/reference/ml/anomaly-detection/ml-configuring-alerts.asciidoc

```diff
@@ -357,7 +357,7 @@ The number of missed documents.
 
 ==== _Error in job messages_
 
-`context.message`::
+`context.message` ^*^::
 A preconstructed message for the rule.
 
 `context.results`::
```

Comment:
For recovered `Error in job messages`, `context.results` is always empty, because there is indeed no error anymore